postgres/server/init.sls

```diff
@@ -76,7 +76,14 @@ postgresql-cluster-prepared:
     - name: {{ postgres.prepare_cluster.command }}
     - unless: {{ postgres.prepare_cluster.test }}
  {%- else %}
-    - name: {{ postgres.prepare_cluster.pgcommand }} {{ postgres.data_dir }}
+    {%- set cc_cmd = '{0} {1}'.format(postgres.prepare_cluster.pgcommand, postgres.data_dir) %}
```

Comment:
Nice. Locale is unset by default. So if value is set thats means locale pillar was set.

Could you move this Jinja logic into `map.jinja` instead of `init.sls` file?